.travis.yml

```diff
@@ -13,7 +13,7 @@ script:
   - ruby ./scripts/image_checker.rb
   - docker run --volume=$(pwd):/app --workdir=/app coala/base:pre /bin/bash -c
     "coala --ci --apply-patches --no-orig;
-    coala-ci || true"
+    coala --ci"
```

Comment:
Wow; it was running twice. Ouch.
So this PR should help solve https://github.com/fossasia/gci16.fossasia.org/issues/226 .